packages/services/f-feature-management-vue/src/createFeatureManagementForVue.js

```diff
@@ -0,0 +1,50 @@
+import createFeatureManagement from '@justeat/f-feature-management';
+import { reactive, set } from 'vue-demi';
+
+/**
+ * Sets a property called `reactive` onto the FeatureManagement proxy, with a `getValue()` function on it.
+ * @param {object} featureManagement The reactive FeatureManagement proxy.
+ */
+function setReactiveProperty (featureManagement) {
+    if (featureManagement) {
+        // set doesn't exist in Vue3, but vue-demi provides a shim
+        set(featureManagement, 'reactive', {
+            getValue: key => featureManagement.getValue(key)
+        });
+    }
+}
+
+/**
+ * Gets a FeatureManagement object, swaps it for a Vue3 proxy (if Vue3 in use),
+ * adds a `reactive` property then updates the `reactive` property every time new config is loaded.
+ * @param {object} settings.  See f-feature-management module docs.
+ * @returns FeatureManagement object ready for use in Vue2/3 as required
+ */
+async function createFeatureManagementForVue (settings) {
+    const originalOnUpdated = settings.onUpdated;
+
+    let featureManagement;
+
+    const overriddenSettings = {
+        ...settings,
+        onUpdated: () => {
+            if (originalOnUpdated && typeof originalOnUpdated === 'function') {
+                originalOnUpdated();
+            }
+            setReactiveProperty(featureManagement);
+        }
+    };
+
+    featureManagement = await createFeatureManagement(overriddenSettings);
+
+    // if Vue2 is being used reactive will be null/undefined
+    if (reactive) {
```

Comment:
So maybe asking the obvious... but is it invalid to try and configure the SDK in a reactive way for Vue 2? I didn't get that impression from the readme

Reply:
Vue 3 reactivity works by swapping out the original object for a proxy, whereas Vue 2 works by adding reactivity behaviour to the object.

So for Vue 3 it's important we get hold of the reactive proxy otherwise we'll be setting properties on the original object, which is of no consequence to Vue.  (Note: Vue tracks the generated proxies i.e. there is only one per object.)

With Vue 2 we simply need to call `Vue.set()` to set properties, and as long as everything is set up for reactivity on that object, changes will be reactive.  But the `reactive()` function is not a thing for Vue2.

Hope that makes sense?

Reply:
You could be ultra-safe here and use the `isVue2` boolean that `vue-demo` offers.

Reply:
Do you think that's safer?  For me it's saying "is there a `reactive` import? is so, use it".  Seems just as safe to me?

Reply:
For anyone not familiar with the differences between both frameworks it would be clearer. Personally, I had no idea that `reactive` would be null if using Vue 2 and would not have known if I wasn\t reviewing this PR!

Reply:
This is true, and is why I wrote the comment on the line above.